crm-sync-api-doc.yaml

```diff
@@ -234,7 +234,7 @@ components:
                   description: Name of the industry. Will update the industry name for every customer that uses the same industry.
                   type: string
             addresses:
-              description: Customer addresses
+              description: Customer addresses. Addresses added here are merged to existing ones (upsert). Existing ones can not be deleted.
```

Comment:
Addresses added through this endpoint are merged with existing ones (upsert). Deleting existing addresses is not supported.